ios/01-query-based-sync/Podfile

```diff
@@ -3,5 +3,5 @@ platform :ios, '11.0'
 
 target 'ToDo' do
   use_frameworks!
-  pod 'RealmSwift', '~> 3.3.0'
+  pod 'RealmSwift'
```

Comment:
This should be `pod 'RealmSwift', '~> 3.13.0'` for both projects, and Podfile.lock should not be committed. We don't want the examples to be pinned to a specific version that will become out of date, but do need to specify a minimum version to ensure that cocoapods doesn't use something ancient if the user hasn't updated the repo recently.